Updates cluster capabilities docs with optional IR disablement (#64481)
openshift/openshift-docs: snippets/capabilities-table.adoc

|`v4.13`
|Specify this option when you want to enable the default capabilities for {product-title} 4.13. By specifying `v4.13`, capabilities that are introduced in newer versions of {product-title} are not enabled. The default capabilities in {product-title} 4.13 are `baremetal`, `marketplace`, `openshift-samples`, `Console`, `Insights`, `Storage`, `CSISnapshot` and `NodeTuning`.

|`v4.14`

> |Specify this option when you want to enable the default capabilities for {product-title} 4.14. By specifying `v4.14`, capabilities that are introduced in newer versions of {product-title} are not enabled. The default capabilities in {product-title} 4.14 are `baremetal`, `marketplace`, `openshift-samples`, `Console`, `Insights`, `Storage`, `CSISnapshot`, `NodeTuning`, and `ImageRegistry`.

Reviewer: ```suggestion
|Specify this option if you want to enable default capabilities for {product-title} 4.14. Specifying `v4.14` does not enable capabilities introduced in newer versions of {product-title}. The default capabilities in {product-title} 4.14 are `baremetal`, `marketplace`, `openshift-samples`, `Console`, `Insights`, `Storage`, `CSISnapshot`, `NodeTuning`, and `ImageRegistry`.

```

Author: Thanks! Going to keep this one as-is, since I recently rewrote it a few days and it had to get approved by various parties. I'll let the installation docs writer decide if he wants to revisit it later.
